apple/bundling/binary_support.bzl

```diff
@@ -27,7 +27,7 @@ def _create_swift_runtime_linkopts_target(
         name,
         deps,
         is_static,
-        tags = None,
+        tags,
```

Comment:
Just because now all consumers use this so we don't need to allow not passing tags. This will at least make any future callers consider if they should pass them

Reply:
If you feel strongly I can obviously add it back

Reply:
`grep -r -C10 \\.add_entitlements_and_swift_linkopts\( apple test` seem to say it all depends on the caller as to if they are passed or not.

Reply:
This specific function is only called privately and both callers of this pass it, even if what they're passing is none

Reply:
I'm confused, I find 12 usages -  https://gist.github.com/thomasvl/f12eb93b963f366b28879d2390e7fb1a

Reply:
I believe you're looking at the wrong function, this is `_create_swift_runtime_linkopts_target` not `add_entitlements_and_swift_linkopts`

Reply:
And I'm doing too many things at once, and looking at the wrong call.  sorry…

Reply:
No worries!